compiler/src/iree/compiler/API/Internal/CompilerDriver.cpp

```diff
@@ -1231,7 +1231,10 @@ void ireeCompilerSetupGlobalCL(int argc, const char **argv, const char *banner,
     llvm::install_out_of_memory_new_handler();
   }
 
-  llvm::cl::ParseCommandLineOptions(argc, argv, banner);
+  llvm::SmallVector<const char *> args(argv, argv + argc);
+  args.push_back("--x86-asm-syntax=intel");
+
+  llvm::cl::ParseCommandLineOptions(args.size(), args.data(), banner);
```

Comment:
maybe can add a comment about LLVM is using AT&T syntax by default, so we need to specify it manually.